### ISSUE TITLE
Migrate blog to github.

### ISSUE THREAD

**Issue by lukehinds** [sigstore/community#170]
Discussed this with @bobcallaway and I think it makes sense to migrate off medium to here. This was we can then use config to allocate approvers (TSC members) and have a review process. 

Anyone wanting to contribute at article can create an issue. 

tagging other TSC folks: @trevrosen @priyawadhwa @SantiagoTorres @bobcallaway

**Comment by trevrosen** [comment]
+1

**Comment by tracymiranda** [comment]
How would you track post analytics on Github?

**Comment by tracymiranda** [comment]
My 2c is that you can still set up a review process for medium in github and allocate approvers.  I'd personally prefer a no-code approach to blog posts.

**Comment by lukehinds** [comment]
We can run analytics ourselves and it does not need to be served from GitHub as such.

I do like medium, but I don't think its a suitable option long term. We might face terms of service changes, questionable takeovers etc.

The other benefit is we can unify the branding with sigstore.dev as well.

**Comment by tracymiranda** [comment]
Yeah medium isn't great - it doesn't really work that well on collaboration too, some aspects of the UX are very painful when trying to get new folks to contribute. 

+1 for unifying branding.
